client/src/components/classesPage/forms/CampaignForm.js

```diff
@@ -215,6 +216,7 @@ function CampaignForm(props) {
   const handleSubmit = e => {
     e.preventDefault();
     props.sendAllToSendgrid(scheduledRefreshrs);
+    handleScheduleRefreshSnack();
```

Comment:
Snackbar looks awesome! It might make sense to move this one to the bottom of the sgDb function in ClassCreateView if we have time. Right now it will pop up as the loading indicator is spinning and the page is blurred.